facia/app/utils/FaciaPicker.scala

```diff
@@ -117,22 +117,28 @@ class FaciaPicker extends GuLogging {
   def getTier(faciaPage: PressedPage)(implicit request: RequestHeader): RenderType = {
     lazy val participatingInTest = ActiveExperiments.isParticipating(DCRFronts)
     lazy val checks = dcrChecks(faciaPage)
+
+    println("checks 1", request.isRss)
+    println("checks 2", checks.get("hasNoPaidForCards"))
+
```

Comment:
Not sure what the convention is here but do we allow print statements in `main` for Frontend